### PR TITLE
CHK-125: Feat/quantity toast message

### PR DESCRIPTION
**Pull request by jeffersontuc**
#### What problem is this solving?

The aim of this PR is to show a toast message when updating the quantity of an item to zero.

We already show the message when removing the item through the delete button.

#### How to test it?

- [Add an item to cart](https://jeff--checkoutio.myvtex.com/cart/add?sku=289)
- Update its quantity to zero
- You should see a toast message saying that the item was removed

#### Screenshots or example usage:

image

### CODE REVIEW

File: react/ProductList.tsx [hunk 1]
```diff
@@ -41,8 +41,14 @@ const ProductList: FunctionComponent<InjectedIntlProps> = ({ intl }) => {
         items: [adjustedItem],
       })
       updateQuantity({ uniqueId, quantity })
+
+      if (quantity === 0) {
+        enqueueToasts([
+          intl.formatMessage(messages.removeToast, { name: item.name }),
+        ])
+      }
```
Comment (lucasecdb) [inline react/ProductList.tsx:49]: what do you think of instead `enqueue`ing the same toast here to call the `handleRemove` and return before updating the quantity?

Comment (wdsrocha) [reply react/ProductList.tsx:49]: Not sure if I really get you, Lucas. It should be before updating the quantity _and_ before the `push` as well, right?

Comment (lucasecdb) [reply react/ProductList.tsx:49]: yep

Comment (jeffersontuc) [reply react/ProductList.tsx:49]: it makes sense! we can do that.

**Review by lucasecdb — APPROVED** [review]
noice ✨

**Review by kaisermann — APPROVED** [review]

**Pull request merged**